src/skills-builder/skills-builder-modal/view-results/ProductTypeBanner.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button, Icon, Stack, useMediaQuery, breakpoints,
+} from '@edx/paragon';
+import { KeyboardArrowDown, KeyboardArrowUp } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import messages from './messages';
+import {
+  DEGREE,
+  BOOT_CAMP,
+  EXECUTIVE_EDUCATION,
+  PROGRAM,
+  COURSE,
+} from './data/constants';
+
+const ProductTypeBanner = ({
+  productTypeName, jobName, numberResults, handleShowAllButtonClick, showAllButtonClickedList,
+}) => {
+  const { formatMessage } = useIntl();
+  const isLarge = useMediaQuery({ minWidth: breakpoints.large.minWidth });
+
+  const normalizeProductTitle = () => {
+    switch (productTypeName) {
+      case COURSE:
+        return 'courses';
+      case BOOT_CAMP:
+        return 'boot camps';
+      case EXECUTIVE_EDUCATION:
+        return 'executive education';
+      case DEGREE:
+        return 'degrees';
+      case PROGRAM:
+        return 'programs';
+      default:
+        return '';
```

Comment:
There may be a better way to do this. But the thinking is that this will give us an easy way to customize the text that is shown for each line of business.

Reply:
Do we have to worry about internationalization?

Reply:
Good find! I'll get these into their own messages 👍